hrui/hrui.php

```diff
@@ -934,3 +944,89 @@ function _hrui_menuSetActive($isActive) {
   CRM_Core_DAO::executeQuery("UPDATE civicrm_navigation SET is_active = {$isActive} WHERE name = 'import_custom_fields'");
   CRM_Core_BAO_Navigation::resetNavigation();
 }
+
+/**
+ * Creates Help Menu in navigation bar
+ *
+ * @param Array $menu List of available menu items
+ */
+function _createHelpMenu(&$menu) {
+  $menu[] = array(
+    'attributes' => array(
+      'label'      => 'Help',
```

Comment:
I mean in the "attribute" part array you pass to `_hrui_civix_insert_navigation_menu`

```

$menu[] = array(
 'attributes' => array(
 'label'      => 'Help', // check if required
 'name'       => 'Help',
 'url'        => null, // check if required
 'permission' => 'access CiviCRM',
 'active'     => 1 // check if required
 )
);
```